### ISSUE TITLE
Not tracking tips and messages are unable to hide.

### ISSUE THREAD

**Issue by VoidLifter** [Semx11/Autotip#34]
![image](https://github.com/Semx11/Autotip/assets/95766705/63ed135b-0076-45fc-b271-f75bcfee9ec4)

i stil seem to be getting the rewards but it really sucks that i cant use stats or hide the messages.
this isue seems to have started since the new booster update.

any progress on this since 2018?
![image](https://github.com/Semx11/Autotip/assets/95766705/1a8b2a72-8e54-4e12-a674-926db8f8beac)

**Comment by VoidLifter** [comment]
Stale issue

**Comment by Semx11** [comment]
yeah sorry dude I don't maintain it anymore. I'm actually surprised it still works. I think @Sk1er maintains it

your main issue is that the message format has changed right? which causes it not to track stats or be able to hide messages?

**Comment by VoidLifter** [comment]
Ah, alright, that might be the issue then.
I checked out @Sk1er but his repo has not been updated since May 4, 2020, so I don't think that repo is being maintained either.
Im not that skilled in Java so although it may sound like a simple fix, I'm not going to attempt fixing it. 🫡

Well, thanks for the reply anyway, sorry to bother you with this sudden message.
